index.bs

```diff
@@ -2227,6 +2227,46 @@ During the above process, the user agent SHOULD show some UI to the user to guid
 authorizing an authenticator. When <code>|options|.{{CredentialCreationOptions/mediation}}</code> is set to {{CredentialMediationRequirement/conditional}}, prominent modal UI should <i>not</i> be shown <i>unless</i> credential creation was previously consented to via means determined by the user agent.
 </div>
 
+#### Registration API Exceptions #### {#sctn-registration-api-exceptions}
```

Comment:
Nit: since this is a subsection of PKC Create, usage of the Create term would be more consistent. "Registration API" almost sounds like its a different API.
```suggestion
#### Create Request Exceptions #### {#sctn-create-request-exceptions}
```

Reply:
I've gone ahead and made this change ✌️